pandas/_testing/contexts.py

```diff
@@ -126,7 +126,7 @@ def ensure_clean(
     handle_or_str: str | IO = str(path)
     if return_filelike:
         kwargs.setdefault("mode", "w+b")
-        handle_or_str = open(path, **kwargs)
+        handle_or_str = open(path, **kwargs, encoding="utf-8")
```

Comment:
apologies, looks like some tests are failing here due to
```python
 TypeError: io.open() got multiple values for keyword argument 'encoding'
```

we may have to do something like
```
handle_or_str = open(path, encoding=kwargs.get('encoding', None), **{key: value for key, value in kwargs.items() if key != 'encoding'})
```
, could you try this please and see if it fixes the failing tests?

Reply:
Ok,I'll do it

Reply:
it run the pre-commit command and i'll commit it.